src/screens/CItyScreen/CityScreen.tsx

```diff
@@ -1,137 +1,58 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
 import { SafeAreaView } from 'react-native-safe-area-context';
-import {
-  ActivityIndicator,
-  Alert,
-  StatusBar,
-  Text,
-  View,
-  ToastAndroid,
-} from 'react-native';
+import { ActivityIndicator, StatusBar, Text, View } from 'react-native';
 import { styles } from './styles';
-import { FAB, Image, Overlay } from 'react-native-elements';
-import { COLORS } from '../../theme/colors';
-import Icon from 'react-native-vector-icons/FontAwesome';
 import { useDispatch, useSelector } from 'react-redux';
 import { WeatherCardDayTemplate } from '../../components/WeatherCardTemplate/WeatherCardTemplate';
-import { dayWeatherInfo } from '../WeatherCardScreen/types';
-import { toggleAppStatus } from '../../store/actions/app';
 import { requestStatus } from '../../store/reducers/appReducer';
-import { toggleDefaultPosition } from '../../store/actions/cities';
-import { selectStatusApp } from '../../store/selectors/appSelector';
 import { getPinnedCities } from '../../store/selectors/citySelector';
-
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const img = require('../../../assets/weather.png');
+import { selectStatusApp } from '../../store/selectors/appSelector';
+import { fetchUsers, weatherGetInfo } from '../../store/sagas/sagasActions';
+import { getDayWeatherInfo } from '../../store/selectors/weatherSelector';
 
 export const CityScreen = () => {
-  const [data, setData] = useState<dayWeatherInfo | null>(null);
-  const [visible, setVisible] = useState(false);
-
   const dispatch = useDispatch();
 
+  const defaultCity = useSelector(getPinnedCities);
   const statusApp = useSelector(selectStatusApp);
-  const pinnedCities = useSelector(getPinnedCities);
+  const data = useSelector(getDayWeatherInfo);
```

Comment:
please try to create more informative names for consts (`weatherInfo` f.e.), use destructuring